src/Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -51,6 +51,13 @@ protected override void HookEvents()
             ItemManipulationModel.FocusSelectedItemsInvoked += ItemManipulationModel_FocusSelectedItemsInvoked;
             ItemManipulationModel.StartRenameItemInvoked += ItemManipulationModel_StartRenameItemInvoked;
             ItemManipulationModel.ScrollIntoViewInvoked += ItemManipulationModel_ScrollIntoViewInvoked;
+            ItemManipulationModel.RefreshItemsThumbnailInvoked += ItemManipulationModel_RefreshItemThumbnail;
+
+        }
+
+        private void ItemManipulationModel_RefreshItemThumbnail(object sender, EventArgs args)
+        {
+            ReloadItemIcons();
```

Comment:
Is there a way to reload the specific icon?

Reply:
Not that I can see. I could try implement it.